lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -914,5 +920,23 @@ describe('StartProcessCloudComponent', () => {
 
             expect(processDropdown).toBeFalsy();
         });
+
+        it('should show the loading spinner before process definitions loaded',()=>{
```

Comment:
please add spaces -> `...loaded', () => { `

Reply:
fixed in new commit